### PR TITLE
fix presentation link

### PR DESCRIPTION
**Pull request by azu**
just a little fixed.

### CODE REVIEW

File: es6/2015-03/mar-24.md [hunk 1]
```diff
@@ -625,7 +625,7 @@ DH: Suggests stage 1, but audit the space which may lead to changes.
 
 Proposal: https://github.com/leebyron/ecmascript-reverse-iterator
 
-LB: [presents slides] (found in github repo)
+LB: [presents slides] \(found in github repo\)
```
Comment (arv) [inline es6/2015-03/mar-24.md]: The intent was to replace `(found in github repo)` with the link above.

Comment (arv) [inline es6/2015-03/mar-24.md]: Actually, the url for the presentation is:

https://github.com/leebyron/ecmascript-reverse-iterable/blob/master/TC39-March2015.pdf

**Comment by azu** [comment]
update url :up:

**Comment by arv** [comment]
Thanks.

**Pull request merged**